src/main/scala/com/kinja/amqp/MessageStore.scala

```diff
@@ -0,0 +1,19 @@
+package com.kinja.amqp
+
+import java.util.UUID
+
+import com.kinja.amqp.model.Message
+import com.kinja.amqp.model.MessageConfirmation
+
+import scala.util.Try
+
+trait MessageStore {
```

Comment:
Make sure that the load and delete actions can be in the same transaction so that different resenders won't collide.